### DISCUSSION TITLE
comelit view

### DISCUSSION THREAD

**Discussion by Stan2402** [briis/unifiprotect#155] [Q&A]
bonjour,
je viens d acheter une maison. elle est équipée dans système de caméra. j'ai 4 cameras. je peux y accéder via une appli ou via une adresse http://. 
est ce que je peux intégrer votre intégration ?

cdlt
stephane

**Comment by briis** [comment]
Sorry, I don't speak French, but I ran your questions through Google Translate, and understand what you are asking. 
This Integration is only for Ubiquity HW, so unless the cameras are of that brand, you cannot use this. Home Assistant has many integrations that support cameras, so try and search for that.

**Comment by Stan2402** [comment]
Thanks for your answersEnvoyé depuis mon téléphone Orange-------- Message original --------De : Bjarne Riis <notifications@github.com>Date : mer. 9 déc. 2020 à 21:49À : briis/unifiprotect <unifiprotect@noreply.github.com>Cc : Stan2402 <Ste.martin2402@gmail.com>, Author <author@noreply.github.com>Objet : Re: [briis/unifiprotect] comelit view (#155)
Sorry, I don't speak French, but I ran your questions through Google Translate, and understand what you are asking.
This Integration is only for Ubiquity HW, so unless the cameras are of that brand, you cannot use this. Home Assistant has many integrations that support cameras, so try and search for that.
